src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/FhirJsonOutputFormatter.cs

```diff
@@ -74,7 +76,10 @@ public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext co
             var elementsSearchParameter = context.HttpContext.GetElementsOrDefault();
             var summarySearchParameter = context.HttpContext.GetSummaryTypeOrDefault();
             var pretty = context.HttpContext.GetPrettyOrDefault();
+            var hasElements = elementsSearchParameter?.Any() == true;
             Resource resource = null;
+            IStructureDefinitionSummaryProvider summaryProvider = new PocoStructureDefinitionSummaryProvider();
```

Comment:
There is an instance of this on `IModelInfoProvider.StructureDefinitionSummaryProvider`